test/functional/fixtures/api/es-next/request-hooks/test.js

```diff
@@ -1,6 +1,19 @@
+const expect = require('chai').expect;
+
 describe('Request Hooks', () => {
-    it('RequestMock', () => {
-        return runTests('./testcafe-fixtures/request-mock.js', 'Basic', { only: 'chrome' });
+    describe('RequestMock', () => {
+        it('Basic', () => {
+            return runTests('./testcafe-fixtures/request-mock/basic.js', 'Basic', { only: 'chrome' });
+        });
+
+        it('Request failed the CORS validation', () => {
+            return runTests('./testcafe-fixtures/request-mock/failed-cors-validation.js', 'Failed CORS validation', { only: 'chrome' })
+                .then(() => {
+                    expect(testReport.warnings).eql([
+                        'RequestMock: CORS validation failed for a request specified as { url: "http://dummy-url.com/get" }'
```

Comment:
Can we report the fixture name and the test name where the warning emitted?

Reply:
For all [warning messages](https://github.com/DevExpress/testcafe/blob/master/src/notifications/warning-message.js) we don't report fixture and test name where the warning was emitted.

We can plan this as an enhancement.
Also the warning message contains stringified request filter rule that cause the warning.